include/zephyr/drivers/fuel_gauge.h

```diff
@@ -115,6 +127,18 @@ struct fuel_gauge_get_property {
 		uint16_t temperature;
 		/** FUEL_GAUGE_VOLTAGE */
 		int voltage;
+		/** FUEL_GAUGE_MODE */
+		uint16_t mode;
+		/** FUEL_GAUGE_CHARGE_CURRENT */
+		uint16_t chg_current;
+		/** FUEL_GAUGE_CHARGE_VOLTAGE */
+		uint16_t chg_voltage;
+		/** FUEL_GAUGE_STATUS */
+		uint16_t status;
```

Comment:
doc build seems to think this has been duplicated

Reply:
Done.